diff -u -r ../emacs.orig/src/editfns.c ./src/editfns.c
--- ../emacs.orig/src/editfns.c 2019-12-26 17:20:12.000000000 +0100
+++ ./src/editfns.c     2019-12-27 14:19:14.204517971 +0100
@@ -1250,7 +1250,7 @@
   if (!pw)
     return Qnil;

-  p = USER_FULL_NAME;
+  p = "unknown";
   /* Chop off everything after the first comma. */
   q = strchr (p, ',');
   full = make_string (p, q ? q - p : strlen (p));